docs/index.md

```diff
@@ -1,91 +1,77 @@
+3. Explore our [Cookbook](./cookbook/index.md): Advanced patterns and real-world applications
 
-This tutorial showcases several key Mirascope features:
+## Key Features
 
-1. **LLM Calls**: The `call` decorator simplifies interactions with LLM APIs.
-2. **Prompt Templates**: Dynamic prompt creation using the `@prompt_template` decorator.
-3. **Custom Tools**: Implementation of web search and content parsing tools.
-4. **Streaming**: Real-time responses using Mirascope's streaming capabilities.
-5. **Agent State Management**: Maintaining conversation history and context, including iteratively calling and inserting tools.
+- LLM Calls: Simplified API interactions across multiple providers
+- Prompt Templates: Dynamic prompt creation with type-safe templating
+- Custom Tools: Extend LLM capabilities with your own functions
+- Streaming: Real-time responses for improved user experience
+- Agent State Management: Maintain conversation context and history effortlessly
 
 ## Next Steps
 
-Once you're comfortable with the tutorial, here are some next steps to deepen your Mirascope knowledge:
-
-### Explore the Documentation
-
-Dive into our comprehensive documentation to learn more about Mirascope's features:
-
-- [Learn Mirascope](./learn/index.md): In-depth guides on all Mirascope features.
-- [API Reference](./api/index.md): Detailed information on Mirascope's classes and functions.
-
-### Try More Examples
-
-Check out our Cookbook for more advanced examples and real-world applications:
-
-- [Mirascope Cookbook](./cookbook/index.md): A collection of recipes for common LLM application patterns.
-
-### Set Up Your Local Environment
-
-To start building your own projects, set up Mirascope in your local environment:
-
-```bash
-pip install mirascope
-pip install "mirascope[openai]"     # when using e.g. mirascope.core.openai
-pip install "mirascope[anthropic]"  # when using e.g. mirascope.core.anthropic
-```
-
-We provide tags for extras for your convenience when using additional modules. These tags will ensure that you install all of the packages necessary for using the modules matching said tags.
-
-### Join the Community
-
-Connect with other Mirascope developers:
-
-- [Slack Community](https://join.slack.com/t/mirascope-community/shared_invite/zt-2ilqhvmki-FB6LWluInUCkkjYD3oSjNA): Get support and share your projects.
-- [GitHub](https://github.com/Mirascope/mirascope): Star us on GitHub, request features, report bugs, and contribute to the project!
+- Read [Why Use Mirascope?](WHY.md) to understand our unique advantages
+- Join our [Slack Community](https://join.slack.com/t/mirascope-community/shared_invite/zt-2ilqhvmki-FB6LWluInUCkkjYD3oSjNA) for support and collaboration
+- Star us on [GitHub](https://github.com/Mirascope/mirascope) to stay updated with the latest developments
 
-We're excited to see what you'll build with Mirascope. Happy coding!
+Ready to revolutionize your LLM development? Let's get started with Mirascope!
```

Comment:
nit: not in love with this last line

what about something like "We're excited to see what you'll build with Mirascope, and we're here to help! Don't hesitate to reach out :)" instead?